hivemq-grafana-dashboard-combined.json

```
@@ -8310,9 +8310,916 @@
+          "title": "MQTT to MongoDB consumer-incoming/s",
+          "type": "timeseries"
+        },
+        {
+          "datasource": {
+            "type": "datasource",
+            "uid": "-- Mixed --"
+          },
+          "description": "The current number of MQTT to MongoDB messages that are in the progress of being forwarded to PostgreSQL.",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "palette-classic"
+              },
+              "custom": {
+                "axisBorderShow": false,
+                "axisCenteredZero": false,
+                "axisColorMode": "text",
+                "axisLabel": "",
+                "axisPlacement": "auto",
+                "axisSoftMin": 0,
+                "barAlignment": 0,
+                "drawStyle": "line",
+                "fillOpacity": 10,
+                "gradientMode": "none",
+                "hideFrom": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": false
+                },
+                "insertNulls": false,
+                "lineInterpolation": "linear",
+                "lineWidth": 1,
+                "pointSize": 5,
+                "scaleDistribution": {
+                  "type": "linear"
+                },
+                "showPoints": "never",
+                "spanNulls": false,
+                "stacking": {
+                  "group": "A",
+                  "mode": "normal"
+                },
+                "thresholdsStyle": {
+                  "mode": "off"
+                }
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": []
+              }
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 7,
+            "w": 6,
+            "x": 18,
+            "y": 69
+          },
+          "id": 118,
+          "interval": "${intervalSeconds}s",
+          "options": {
+            "legend": {
+              "calcs": [],
+              "displayMode": "list",
+              "placement": "bottom",
+              "showLegend": false
+            },
+            "tooltip": {
+              "mode": "multi",
+              "sort": "none"
+            }
+          },
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "${Prometheus}"
+              },
+              "editorMode": "code",
+              "expr": "label_replace({__name__=~\"com_hivemq_extensions_mongodb_mqtt_to_mongodb_(.+)_in_progress_count_current\", __name__!=\"com_hivemq_extensions_mongodb_mqtt_to_mongodb_total_in_progress_count_current\", __name__!~\"com_hivemq_extensions_mongodb_mqtt_to_mongodb_(.+)batches_in_progress_count_current\"}, \"route\", \"$1\", \"__name__\", \"com_hivemq_extensions_mongodb_mqtt_to_mongodb_(.+)_in_progress_count_current\")",
```

```suggestion
              "expr": "label_replace({__name__=~\"com_hivemq_extensions_mongodb_mqtt_to_mongodb_(.+)_in_progress_count_current\", __name__!=\"com_hivemq_extensions_mongodb_mqtt_to_mongodb_total_in_progress_count_current\"}, \"route\", \"$1\", \"__name__\", \"com_hivemq_extensions_mongodb_mqtt_to_mongodb_(.+)_in_progress_count_current\")",
```

we don't have a batches in progress metric on mongodb